### PR TITLE
Fix: Case mismatch

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] fixes case mismatches in class and method name references

### CODE REVIEW

File: tests/Facebook/InstantArticles/Elements/CiteTest.php [hunk 1]
```diff
@@ -147,7 +147,7 @@ public function testRenderWithLink()
                 ->appendText('Some ')
                 ->appendText(
                     Anchor::create()
-                        ->withHRef('http://foo.com')
+                        ->withHref('http://foo.com')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/CiteTest.php]: See [`Anchor::withHref()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Anchor.php#L47-L52).

File: tests/Facebook/InstantArticles/Elements/H1Test.php [hunk 2]
```diff
@@ -130,7 +130,7 @@ public function testRenderWithLink()
                 ->appendText('Some ')
                 ->appendText(
                     Anchor::create()
-                        ->withHRef('http://foo.com')
+                        ->withHref('http://foo.com')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/H1Test.php]: See [`Anchor::withHref()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Anchor.php#L47-L52).

File: tests/Facebook/InstantArticles/Elements/FooterTest.php [hunk 3]
```diff
@@ -113,7 +113,7 @@ public function testRenderWithCopyrightSmallElement()
                 ->appendText("2016 ")
                 ->appendText(
                     Anchor::create()
-                        ->withHRef('https://facebook.com')
+                        ->withHref('https://facebook.com')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/FooterTest.php]: See [`Anchor::withHref()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Anchor.php#L47-L52).

File: tests/Facebook/InstantArticles/Elements/H2Test.php [hunk 4]
```diff
@@ -130,7 +130,7 @@ public function testRenderWithLink()
                 ->appendText('Some ')
                 ->appendText(
                     Anchor::create()
-                        ->withHRef('http://foo.com')
+                        ->withHref('http://foo.com')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/H2Test.php]: See [`Anchor::withHref()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Anchor.php#L47-L52).

File: tests/Facebook/InstantArticles/Elements/H3Test.php [hunk 5]
```diff
@@ -130,7 +130,7 @@ public function testRenderWithLink()
                 ->appendText('Some ')
                 ->appendText(
                     Anchor::create()
-                        ->withHRef('http://foo.com')
+                        ->withHref('http://foo.com')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/H3Test.php]: See [`Anchor::withHref()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Anchor.php#L47-L52).

File: tests/Facebook/InstantArticles/Elements/ParagraphTest.php [hunk 6]
```diff
@@ -86,7 +86,7 @@ public function testRenderWithLink()
                 ->appendText('Some ')
                 ->appendText(
                     Anchor::create()
-                        ->withHRef('http://foo.com')
+                        ->withHref('http://foo.com')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/ParagraphTest.php]: See [`Anchor::withHref()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Anchor.php#L47-L52).

File: src/Facebook/InstantArticles/Elements/AnimatedGIF.php [hunk 7]
```diff
@@ -15,7 +15,7 @@
  *     <li>Audio</li>
 *      <li>Image</li>
  *     <li>Video</li>
- *     <li>SlideShow</li>
+ *     <li>Slideshow</li>
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/AnimatedGIF.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/AnimatedGIF.php [hunk 8]
```diff
@@ -28,7 +28,7 @@
  * @see Audio
  * @see Image
  * @see Video
- * @see SlideShow
+ * @see Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/AnimatedGIF.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Audible.php [hunk 9]
```diff
@@ -13,7 +13,7 @@
  * This class represents all elements that can contain Audio
  * <ul>
  *     <li>Image</li>
- *     <li>SlideShow</li>
+ *     <li>Slideshow</li>
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Audible.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Audible.php [hunk 10]
```diff
@@ -22,7 +22,7 @@
  *  </audio>
  *
  * @see Image
- * @see SlideShow
+ * @see Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Audible.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Audio.php [hunk 11]
```diff
@@ -16,7 +16,7 @@
  * <ul>
  *     <li>Image</li>
  *     <li>Video</li>
- *     <li>SlideShow</li>
+ *     <li>Slideshow</li>
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Audio.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Audio.php [hunk 12]
```diff
@@ -27,7 +27,7 @@
  *
  * @see Image
  * @see Video
- * @see SlideShow
+ * @see Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Audio.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Caption.php [hunk 13]
```diff
@@ -16,7 +16,7 @@
  * <ul>
  *     <li>Image</li>
  *     <li>Video</li>
- *     <li>SlideShow</li>
+ *     <li>Slideshow</li>
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Caption.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Caption.php [hunk 14]
```diff
@@ -29,7 +29,7 @@
  *
  * @see Image
  * @see Video
- * @see SlideShow
+ * @see Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Caption.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Image.php [hunk 15]
```diff
@@ -17,7 +17,7 @@
  * <ul>
  *     <li>Audio</li>
  *     <li>Video</li>
- *     <li>SlideShow</li>
+ *     <li>Slideshow</li>
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Image.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Image.php [hunk 16]
```diff
@@ -29,7 +29,7 @@
  *
  * @see Audio
  * @see Video
- * @see SlideShow
+ * @see Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Image.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Slideshow.php [hunk 17]
```diff
@@ -11,7 +11,7 @@
 use Facebook\InstantArticles\Validators\Type;
 
 /**
- * Class SlideShow
+ * Class Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Slideshow.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Video.php [hunk 18]
```diff
@@ -17,7 +17,7 @@
  * <ul>
  *     <li>Audio</li>
  *     <li>Image</li>
- *     <li>SlideShow</li>
+ *     <li>Slideshow</li>
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Video.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: src/Facebook/InstantArticles/Elements/Video.php [hunk 19]
```diff
@@ -30,7 +30,7 @@
  *
  * @see Audio
  * @see Image
- * @see SlideShow
+ * @see Slideshow
```
Comment (localheinz) [inline src/Facebook/InstantArticles/Elements/Video.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/HeaderTest.php [hunk 20]
```diff
@@ -232,7 +232,7 @@ public function testHeaderWithSlideshow()
                         ->appendText(Bold::create()->appendText('in Bold'))
                 )
                 ->withCover(
-                    SlideShow::create()
+                    Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/HeaderTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/InstantArticleTest.php [hunk 21]
```diff
@@ -94,7 +94,7 @@ protected function setUp()
 
                 // Slideshow
                 ->addChild(
-                    SlideShow::create()
+                    Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/InstantArticleTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/InstantArticleTest.php [hunk 22]
```diff
@@ -24,7 +24,7 @@ protected function setUp()
 
         $this->article =
             InstantArticle::create()
-                ->withCanonicalUrl('http://foo.com/article.html')
+                ->withCanonicalURL('http://foo.com/article.html')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/InstantArticleTest.php]: See [`InstantArticle::withCanonicalURL()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/c0d9491ad7c44ae1e96fa7d5f7f54262b9f8186d/src/Facebook/InstantArticles/Elements/InstantArticle.php#L117-L123).

File: tests/Facebook/InstantArticles/Elements/InstantArticleTest.php [hunk 23]
```diff
@@ -286,7 +286,7 @@ public function testInstantArticleAlmostEmpty()
     {
         $article =
             InstantArticle::create()
-                ->withCanonicalUrl('')
+                ->withCanonicalURL('')
                 ->withHeader(Header::create())
                 // Paragraph1
                 ->addChild(
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/InstantArticleTest.php]: See [`InstantArticle::withCanonicalURL()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/c0d9491ad7c44ae1e96fa7d5f7f54262b9f8186d/src/Facebook/InstantArticles/Elements/InstantArticle.php#L117-L123).

File: tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php [hunk 24]
```diff
@@ -28,7 +28,7 @@ public function testInstantArticle()
         $article =
             InstantArticle::create()
                 // Warning 1 - Invalid canonicalURL
-                ->withCanonicalUrl('')
+                ->withCanonicalURL('')
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php]: See [`InstantArticle::withCanonicalURL()`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/c0d9491ad7c44ae1e96fa7d5f7f54262b9f8186d/src/Facebook/InstantArticles/Elements/InstantArticle.php#L117-L123).

File: tests/Facebook/InstantArticles/Elements/InstantArticleTest.php [hunk 25]
```diff
@@ -318,7 +318,7 @@ public function testInstantArticleAlmostEmpty()
 
                 // Slideshow
                 ->addChild(
-                    SlideShow::create()
+                    Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/InstantArticleTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/SlideshowTest.php [hunk 26]
```diff
@@ -12,7 +12,7 @@ class SlideshowTest extends \PHPUnit_Framework_TestCase
 {
     public function testRenderEmpty()
     {
-        $slideshow = SlideShow::create();
+        $slideshow = Slideshow::create();
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/SlideshowTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/SlideshowTest.php [hunk 27]
```diff
@@ -23,7 +23,7 @@ public function testRenderEmpty()
     public function testRenderBasic()
     {
         $slideshow =
-            SlideShow::create()
+            Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/SlideshowTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/SlideshowTest.php [hunk 28]
```diff
@@ -48,7 +48,7 @@ public function testRenderBasic()
     public function testRenderWithLikeAndComments()
     {
         $slideshow =
-            SlideShow::create()
+            Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/SlideshowTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/SlideshowTest.php [hunk 29]
```diff
@@ -86,7 +86,7 @@ public function testRenderWithLikeAndComments()
     public function testRenderWithCaption()
     {
         $slideshow =
-            SlideShow::create()
+            Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/SlideshowTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/SlideshowTest.php [hunk 30]
```diff
@@ -132,7 +132,7 @@ public function testRenderWithGeotag()
 JSON;
 
         $slideshow =
-            SlideShow::create()
+            Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/SlideshowTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/SlideshowTest.php [hunk 31]
```diff
@@ -173,7 +173,7 @@ public function testRenderWithAudio()
             '</audio>';
 
         $slideshow =
-            SlideShow::create()
+            Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/SlideshowTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php [hunk 32]
```diff
@@ -14,7 +14,7 @@
 use Facebook\InstantArticles\Elements\Header;
 use Facebook\InstantArticles\Elements\Footer;
 use Facebook\InstantArticles\Elements\Paragraph;
-use Facebook\InstantArticles\Elements\SlideShow;
+use Facebook\InstantArticles\Elements\Slideshow;
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php [hunk 33]
```diff
@@ -55,7 +55,7 @@ public function testInstantArticle()
 
                 // Slideshow
                 ->addChild(
-                    SlideShow::create()
+                    Slideshow::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/Validators/InstantArticleValidatorTest.php]: See [`Slideshow`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/46a5cded2c705fcbe60bbf6e22fc7f648a6ff6e2/src/Facebook/InstantArticles/Elements/Slideshow.php#L32-L290).

File: tests/Facebook/InstantArticles/Elements/InteractiveTest.php [hunk 34]
```diff
@@ -44,7 +44,7 @@ public function testRenderBasic()
     public function testRenderBasicWithCaption()
     {
         $social_embed =
-            interactive::create()
+            Interactive::create()
```
Comment (localheinz) [inline tests/Facebook/InstantArticles/Elements/InteractiveTest.php]: See [`Interactive`](https://github.com/facebook/facebook-instant-articles-sdk-php/blob/c0d9491ad7c44ae1e96fa7d5f7f54262b9f8186d/src/Facebook/InstantArticles/Elements/Interactive.php#L24-L270).

**Comment by everton-rosario** [comment]
Unfortunately these are breaking changes, that we might not take on.

**Comment by timjacobi** [comment]
@everton-rosario can you elaborate? Check are passing.

**Comment by localheinz** [comment]
Thank you, @everton-rosario!

**Pull request merged**